### PR TITLE
Migration: OfflineExporting

### PR DESCRIPTION
**Pull request by marvin19**
Improved OfflineExporting.

### CODE REVIEW

File: ts/Core/Globals.ts [hunk 1]
```diff
@@ -83,6 +82,11 @@ declare global {
     interface TouchList {
         changedTouches: Array<Touch>;
     }
+    interface Window {
+        canvg: CanvgNamespace;
+        jsPDF: typeof jsPDF;
+        svg2pdf: Function;
+    }
```
Comment (bre1470) [inline ts/Core/Globals.ts:89]: This should go into the Vendor declarations too.

File: ts/Core/Globals.ts [hunk 2]
```diff
@@ -91,6 +95,7 @@ declare global {
     const win: Window|undefined;
 }
 
+
```
Comment (bre1470) [inline ts/Core/Globals.ts:98]: Double space.

File: ts/Extensions/OfflineExporting/OfflineExporting.ts [hunk 3]
```diff
@@ -0,0 +1,905 @@
...
+ *  (c) 2015 Torstein Honsi / Oystein Moseng
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* global MSBlobBuilder */
+
```
Comment (bre1470) [inline ts/Extensions/OfflineExporting/OfflineExporting.ts:14]: `'use strict';` missing

File: ts/Extensions/OfflineExporting/OfflineExporting.ts [hunk 4]
```diff
@@ -0,0 +1,905 @@
...
+    /**
+     * Compose
+     *
+     * @private
+     *
+     * @param ChartClass
+     * Axis class to extend.
+     */
+
+    export function compose<T extends typeof Chart>(
```
Comment (bre1470) [inline ts/Extensions/OfflineExporting/OfflineExporting.ts:134]: Doclets without space. Section comments with space.

File: ts/masters/modules/exporting.src.ts [hunk 5]
```diff
@@ -14,9 +14,11 @@ import Highcharts from '../../Core/Globals.js';
...
 import Exporting from '../../Extensions/Exporting/Exporting.js';
 import HttpUtilities from '../../Core/HttpUtilities.js';
+import OfflineExporting from '../../Extensions/OfflineExporting/OfflineExporting.js';
 const G: AnyRecord = Highcharts;
 G.HttpUtilities = HttpUtilities;
 G.ajax = HttpUtilities.ajax;
 G.getJSON = HttpUtilities.getJSON;
 G.post = HttpUtilities.post;
 Exporting.compose(G.Chart, G.Renderer);
+OfflineExporting.compose(G.Chart);
```
Comment (bre1470) [inline ts/masters/modules/exporting.src.ts:24]: Offline exporting is not part of exporting.

**Review by bre1470 — CHANGES_REQUESTED** [review]

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**